app/src/main/java/com/monkeyteam/chimpagne/ui/HomeScreen.kt

```diff
@@ -67,22 +67,6 @@ import com.monkeyteam.chimpagne.ui.utilities.PromptLogin
 import com.monkeyteam.chimpagne.viewmodels.AccountViewModel
 import com.monkeyteam.chimpagne.viewmodels.FindEventsViewModel
 
-class LocationViewModel(myContext: Context) {
-  private val fusedLocationClient = LocationServices.getFusedLocationProviderClient(myContext)
-
-  fun startLocationUpdates(
-      myContext: Context,
-      onLocationSuccess: (lat: Double, lng: Double) -> Unit
-  ) {
-    if (ActivityCompat.checkSelfPermission(myContext, Manifest.permission.ACCESS_FINE_LOCATION) ==
```

Comment:
ACCESS_FINE_LOCATION should be asked with ACCESS_COARSE_LOCATION as discibed in this documentation page :

[here](https://developer.android.com/develop/sensors-and-location/location/permissions?hl=fr#foreground)